src/test/cli-integration/rbd/snap-diff.t

```diff
@@ -0,0 +1,47 @@
+  $ ceph osd pool create xrbddiff1 8
```

Comment:
@smithfarm I still see 'application not enabled on 1 pool' theuthology failures [1]. I think you have to add `rbd pool init xrbddiff1` after the pool creation to avoid the warning.

[1] http://pulpito.ceph.com/trociny-2019-06-14_04:21:54-rbd-wip-mgolub-testing-distro-basic-smithi/

Reply:
@trociny Thanks - added it.